versions.gradle

```diff
@@ -24,15 +24,15 @@ ext.deps = [:]
 def versions = [:]
 // GeckoView versions can be found here:
 // https://maven.mozilla.org/?prefix=maven2/org/mozilla/geckoview/
-versions.gecko_view_nightly = "115.0.20230605094751"
-versions.gecko_view_beta = "114.0.20230525180143"
-versions.android_components = "116.0b2"
+versions.gecko_view_nightly = "105.0.20220822095220"
+versions.gecko_view_beta = "104.0.20220816121120"
+versions.android_components = "105.0.8"
```

Comment:
Same here, let's merge all these commits together to avoid the back and forth

Reply:
Note that I perfectly understand that the different commits tell a story about how this was fixed, but we should rework it before landing.